contracts.go

```diff
@@ -1,109 +1,109 @@
 package main
 
 import (
-	"bytes"
-	"crypto/sha256"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"path/filepath"
-	"regexp"
-	"strings"
-	"text/template"
-
-	"github.com/spf13/cobra"
+  "bytes"
+  "crypto/sha256"
+  "fmt"
+  "io/ioutil"
+  "os"
+  "os/exec"
+  "path/filepath"
+  "regexp"
+  "strings"
+  "text/template"
+
+  "github.com/spf13/cobra"
```

Comment:
do you have your editor set for another formatting type? set `gofmt` in your editor; go is very strict about code formatting